bindings/nodejs/generated.d.ts

```diff
@@ -265,21 +275,23 @@ export class Operator {
    * await op.write("path/to/file", Buffer.from("hello world"));
    * // or
    * await op.write("path/to/file", "hello world");
+   * // or
+   * await op.write("path/to/file", Buffer.from("hello world"), { contentType: "text/plain" });
    * ```
    */
-  write(path: string, content: Buffer | string): Promise<void>
+  write(path: string, content: Buffer | string, options?: WriteWith | undefined | null): Promise<void>
   /**
    * Write multiple bytes into path.
    *
    * It could be used to write large file in a streaming way.
    */
-  writer(path: string): Promise<Writer>
+  writer(path: string, options?: WriteWith | undefined | null): Promise<Writer>
```

Comment:
`writer` might have different options with `wirte`, so we can accept `WriterOptions` here.

Reply:
Currently, this `opWriteOptions` corresponds to `OpWrite`, so I think there is no need for a `WriterOptions`